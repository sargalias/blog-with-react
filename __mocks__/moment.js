import moment from 'moment';

const mockMoment = (date=0) => moment(date);

export default mockMoment;
