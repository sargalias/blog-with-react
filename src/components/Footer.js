import React from 'react';
import moment from 'moment';

const Footer = () => (
  <div>
    <div class="mt-5"></div>
    <footer class="bg-dark">
      <div class="container">
        <div id="copyright">&copy; Spyros Argalias {moment().format('YYYY')}</div>
        <div id="website-link">App by <a href="https://sargalias.com" target="_blank">Spyros Argalias</a></div>
        <div id="code-repository-link"><a href="https://github.com/sargalias/blog-with-react" target="_blank">View code on GitHub</a></div>
      </div>
    </footer>
  </div>
);

export default Footer;
