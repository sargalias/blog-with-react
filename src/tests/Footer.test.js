import React from 'react';
import { shallow } from 'enzyme';
import Footer from '../components/Footer';
import moment from 'moment';

test('Footer should render correctly', () => {
  const wrapper = shallow(<Footer />);
  expect(wrapper).toMatchSnapshot();
});
