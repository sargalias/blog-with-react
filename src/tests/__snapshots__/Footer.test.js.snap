// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render correctly 1`] = `
<div>
  <div
    class="mt-5"
  />
  <footer
    class="bg-dark"
  >
    <div
      class="container"
    >
      <div
        id="copyright"
      >
        Â© Spyros Argalias 
        1970
      </div>
      <div
        id="website-link"
      >
        App by 
        <a
          href="https://sargalias.com"
          target="_blank"
        >
          Spyros Argalias
        </a>
      </div>
      <div
        id="code-repository-link"
      >
        <a
          href="https://github.com/sargalias/blog-with-react"
          target="_blank"
        >
          View code on GitHub
        </a>
      </div>
    </div>
  </footer>
</div>
`;
